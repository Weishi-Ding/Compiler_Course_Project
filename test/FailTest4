class Animal {
    constructor Animal(){

    }
}


int main(){
    Animal cat = new Animal();

    cat.getName();
}