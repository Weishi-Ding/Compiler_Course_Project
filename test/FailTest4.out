Fatal error: exception Failure("Function getName is undefined")